src/components/Images/Images.js

```diff
@@ -51,13 +171,24 @@ CloudinaryImage.CROP_METHODS = {
   CROP: 'crop',
 }
 
-CloudinaryImage.propTypes = {
-  // We're trying to avoid/reduce className restyling, but let's try it here:
+CloudinaryImage.IMAGE_FILE_TYPES = {
```

Comment:
Non Blocking:
My sense is we'll eventually do the same separating of concerns for image types e.g. invert the control. But I don't have a clear recommendation for how to do that yet.